### PR TITLE
en: add a simpe desc about HA

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

- add a simpe desc about HA

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/322
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/_index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ aliases: ['/docs/tidb-data-migration/dev/']
 
 **What's new in DM v2.0:**
 
-- High availability. The data migration task can run normally even when some DM-master or DM-worker nodes fail.
+- [High availability of data migration task](overview.md#high-availability). The data migration task can run normally even when some DM-master or DM-worker nodes fail.
```
Comment (yikeke) [inline en/_index.md:13]:
```suggestion
- [High availability of data migration tasks](overview.md#high-availability). The data migration task can run normally even when some DM-master or DM-worker nodes fail.
```

File: en/overview.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ aliases: ['/docs/tidb-data-migration/dev/overview/']
 
 **What's new in DM v2.0:**
 
-- High availability. The data migration task can run normally even when some DM-master or DM-worker nodes fail.
+- [High availability of data migration task](overview.md#high-availability). The data migration task can run normally even when some DM-master or DM-worker nodes fail.
```
Comment (yikeke) [inline en/overview.md:15]:
```suggestion
- [High availability of data migration task](#high-availability). The data migration task can run normally even when some DM-master or DM-worker nodes fail.
```

File: en/overview.md [hunk 3]
```diff
@@ -58,6 +58,12 @@ dmctl is the command line tool used to control the DM cluster.
 - Handling the errors during data migration tasks
 - Verifying the configuration correctness of data migration tasks
 
+### High availability
+
+When multiple DM-master nodes are deployed, all DM-master nodes will use the embedded etcd to form a cluster. The DM-master cluster is used to store metadata such as cluster node information and task configuration. The leader node elected through etcd is used to provide services related to cluster management and data migration task management. Therefore, if the number of available DM-master nodes exceeds half of the deployed nodes, services can be provided normally.
```
Comment (yikeke) [inline en/overview.md:63]:
```suggestion
When you deploy multiple DM-master nodes, all DM-master nodes use the embedded etcd to form a cluster. The DM-master cluster is used to store metadata such as cluster node information and task configuration. The leader node elected through etcd is used to provide services such as cluster management and data migration task management. Therefore, if the number of available DM-master nodes exceeds half of the deployed nodes, the DM cluster can normally provide services.
```
Use active voice and present tense

File: en/overview.md [hunk 4]
```diff
@@ -58,6 +58,12 @@ dmctl is the command line tool used to control the DM cluster.
...
+
+When the number of deployed DM-worker nodes exceeds the number of upstream MySQL/MariaDB nodes, the related DM-worker nodes that exceed the number of upstream nodes will be idle by default. If a DM-worker node goes offline or is isolated from the DM-master leader, the DM-master can automatically schedule data migration tasks of the original DM-worker node to other idle DM-worker nodes. (If the worker node is isolated, the related data migration tasks will be automatically stopped); if no idle DM-worker nodes can be scheduled, the data migration tasks of the original DM-worker will not be performed.
```
Comment (yikeke) [inline en/overview.md:65]:
```suggestion
When the number of deployed DM-worker nodes exceeds the number of upstream MySQL/MariaDB nodes, the extra DM-worker nodes are idle by default. If a DM-worker node goes offline or is isolated from the DM-master leader, DM-master automatically schedules data migration tasks of the original DM-worker node to other idle DM-worker nodes. (If a DM-worker node is isolated, it automatically stops the data migration tasks on it); if there are no available idle DM-worker nodes, the data migration tasks of the original DM-worker cannot be performed.
```

Comment (yikeke) [reply en/overview.md:65]: PTAL at my suggestion, especially this one: `If a DM-worker node is isolated, it automatically stops the data migration tasks on it` @csuzhangxc

Comment (csuzhangxc) [reply en/overview.md:65]: your suggestion is right. 👍

**Comment by csuzhangxc** [comment]
@ireneontheway please address https://github.com/pingcap/docs-dm/pull/366#discussion_r485476523. rest LGTM

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**